README.md

```diff
@@ -1,5 +1,5 @@
 
-# <img src="https://github.com/JosefNemec/Playnite/raw/master/web/applogo.png" width="32">  Playnite
+# <img src="https://github.com/JosefNemec/Playnite/raw/master/web/applogo.png" width="32">  Playnite ![Build](https://github.com/JosefNemec/Playnite/workflows/Build/badge.svg)
```

Comment:
This will look something like the following when it's up and running:

![image](https://user-images.githubusercontent.com/1100550/105640307-3a095d00-5e32-11eb-80dd-ecb5ad0cc845.png)